src/panel.js

```diff
@@ -153,8 +156,9 @@ $(document).ready(() => {
   state.update('destination/marker', flag)
 })
 
-// ----
+// .............................................................................................
 // ui elements setup
+dom.linkDemo.on('click', () => chrome.tabs.update({ url: 'https://codepen.io/cobi-bike/pen/VzBOqp?editors=0010' }))
```

Comment:
out of curiosity, shouldn't this type of string variables be in a config file to avoid using magic variables?

Reply:
yes they should be, but if I start doing that in this PR then it would be huge. I agree on doing this but I think we should do it as part of another task